packages/frontend/src/pages/Home.tsx

```diff
@@ -78,7 +81,7 @@ const Home: FC = () => {
                             projectUuid={project.data.projectUuid}
                         />
                         <PinnedItemsPanel
-                            data={{ dashboards, savedCharts, spaces }}
+                            data={pinnedItems!}
```

Comment:
It would be better if we set the default data instead of forcing the defined type:
```const { data: pinnedItems = { dashboards: [], charts: [], spaces: []}, isLoading: pinnedItemsLoading }```